#Definition of the Classification Output type
#the matched class
string className
#the membership value
float64 membership
